packages/core/src/indexing-store/cache.ts

```diff
@@ -347,10 +347,13 @@ export const createIndexingCache = ({
         updateBuffer.get(table)!.delete(getCacheKey(table, key));
 
       if (
-        cache.get(table)!.has(getCacheKey(table, key)) &&
-        cache.get(table)!.get(getCacheKey(table, key))
+        (cache.get(table)!.has(getCacheKey(table, key)) &&
```

Comment:
Could we do `const cacheKey = getCacheKey(table, key)` once at the top of this function? `getCacheKey` seems quite expensive.

Also, isn't it redundant to do `Map.has(val) && Map.get(val)` to get a boolean?

Reply:
The `get` may return `null`